tests/test_contact.py

```diff
@@ -43,20 +44,23 @@ def test_de_json_required(self, bot):
 
     def test_de_json_all(self, bot):
         json_dict = {'phone_number': self.phone_number, 'first_name': self.first_name,
-                     'last_name': self.last_name, 'user_id': self.user_id}
+                     'last_name': self.last_name, 'user_id': self.user_id,
+                     'vcard': self.vcard}
         contact = Contact.de_json(json_dict, bot)
 
         assert contact.phone_number == self.phone_number
         assert contact.first_name == self.first_name
         assert contact.last_name == self.last_name
         assert contact.user_id == self.user_id
+        assert contact.vcard == self.vcard
 
     def test_send_with_contact(self, monkeypatch, bot, chat_id, contact):
         def test(_, url, data, **kwargs):
             phone = data['phone_number'] == contact.phone_number
             first = data['first_name'] == contact.first_name
             last = data['last_name'] == contact.last_name
-            return phone and first and last
+            vcard = data['vcard'] == contact.vcard
```

Comment:
I think this should split int two seperate tests. One with vcard, one without. Also see comment above